### PR TITLE
Get current state route

### PR DESCRIPTION
**Pull request by Lucasdvs10**

### CODE REVIEW

File: src/modules/get_current_order_state_by_id/app/get_current_order_state_by_id_controller.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from src.modules.get_current_order_state_by_id.app.get_current_order_state_by_id_usecase import \
+    GetCurrentOrderStateByIdUsecase
+from src.modules.get_current_order_state_by_id.app.get_current_order_state_by_id_viewmodel import \
```
Comment (VgsStudio) [inline src/modules/get_current_order_state_by_id/app/get_current_order_state_by_id_controller.py]: import

File: src/modules/get_current_order_state_by_id/app/get_current_order_state_by_id_presenter.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+from src.modules.get_current_order_state_by_id.app.get_current_order_state_by_id_controller import \
+    GetCurrentOrderStateByIdController
+from src.modules.get_current_order_state_by_id.app.get_current_order_state_by_id_usecase import \
```
Comment (VgsStudio) [inline src/modules/get_current_order_state_by_id/app/get_current_order_state_by_id_presenter.py]: import

File: src/modules/get_current_order_state_by_id/app/get_current_order_state_by_id_presenter.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    GetCurrentOrderStateByIdUsecase
+from src.shared.environments import Environments
+from src.shared.helpers.external_interfaces.http_lambda_requests import LambdaHttpRequest, LambdaHttpResponse
+
+repo_order = Environments.get_order_repo()()
+repo_product = Environments.get_product_repo()()
```
Comment (VgsStudio) [inline src/modules/get_current_order_state_by_id/app/get_current_order_state_by_id_presenter.py]: tira esse repo

**Review by VgsStudio — CHANGES_REQUESTED** [review]

**Comment by VgsStudio** [comment]
adiciona no LAMBDA stack

File: src/modules/get_current_order_state_by_id/app/get_current_order_state_by_id_controller.py (hunk 1 above)
Comment (Lucasdvs10) [reply src/modules/get_current_order_state_by_id/app/get_current_order_state_by_id_controller.py]: OK

File: src/modules/get_current_order_state_by_id/app/get_current_order_state_by_id_presenter.py (hunk 2 above)
Comment (Lucasdvs10) [reply src/modules/get_current_order_state_by_id/app/get_current_order_state_by_id_presenter.py]: OK

File: src/modules/get_current_order_state_by_id/app/get_current_order_state_by_id_presenter.py (hunk 3 above)
Comment (Lucasdvs10) [reply src/modules/get_current_order_state_by_id/app/get_current_order_state_by_id_presenter.py]: Ok

**Pull request merged**